### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by ivan-dalmet**

### CODE REVIEW

File: src/app/account/account.types.ts [hunk 1]
```diff
@@ -1,3 +0,0 @@
-import { User } from '@/app/admin/users/users.types';
-
-export type Account = User;
```
Comment (Jessy-BAER) [inline src/app/account/account.types.ts:3]: We want to keep this empty file ?

File: src/spa/account/PageRegister.tsx [hunk 2]
```diff
@@ -190,11 +190,15 @@ export const PageRegister = () => {
                 validations={[
                   {
                     rule: isMinLength(4),
-                    message: t('account:data.password.tooShort', { min: 4 }),
+                    message: String(
```
[the remark this answers is not in the document]
Comment (ivan-dalmet) [reply src/spa/account/PageRegister.tsx:193]: No I will revert it :)

File: src/types/i18next.d.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import 'i18next';
+
+declare module 'i18next' {
+  interface CustomTypeOptions {
+    returnNull: false;
+  }
```
[the remark this answers is not in the document]
Comment (ivan-dalmet) [reply src/types/i18next.d.ts:6]: Yes I will add the link to the issue or documentation

File: src/app/account/account.types.ts (hunk 1 above)
Comment (ivan-dalmet) [reply src/app/account/account.types.ts:3]: I don't remember why I removed this 😅 
I will either remove the file or revert the change

File: src/spa/account/PageRegister.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ivan-dalmet) [reply src/spa/account/PageRegister.tsx]: I reverted some other `String(` added on translation

File: src/app/account/account.types.ts (hunk 1 above)
Comment (ivan-dalmet) [reply src/app/account/account.types.ts:3]: Oh it just that the file was moved from `app` to `spa`

File: src/types/i18next.d.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (ivan-dalmet) [reply src/types/i18next.d.ts:8]: done

**Review by Jessy-BAER — APPROVED** [review]

**Review by DecampsRenan — APPROVED** [review]

**Pull request merged**